libyul/AsmAnalysis.cpp

```diff
@@ -553,6 +556,16 @@ bool AsmAnalyzer::warnOnInstructions(evmasm::Instruction _instr, SourceLocation
 		errorForVM("only available for Constantinople-compatible");
 	else if (_instr == evmasm::Instruction::CHAINID && !m_evmVersion.hasChainID())
 		errorForVM("only available for Istanbul-compatible");
+	else if (_instr == evmasm::Instruction::PC)
+	{
```

Comment:
You can remove those `{}` here.

Reply:
the coding guidelines say "should", but sure I can :-)